### PR TITLE
Add CODEOWNERS file

### PR DESCRIPTION
**Pull request by cbaatz-da**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+/daml-lf/           @digital-asset/daml-lf
+/ledger/            @digital-asset/ledger
+/ledger-api/        @digital-asset/ledger
+
+# Ecosystems
+/language-support/          @digital-asset/java @digital-asset/scala @digital-asset/js
+/language-support/java/     @digital-asset/java
+/language-support/js/       @digital-asset/js
+/language-support/scala/    @digital-asset/scala
+/scala-protoc-plugins/      @digital-asset/scala
```
Comment (gerolf-da) [inline CODEOWNERS]: @cbaatz-da: could you add `@digital-asset/ledger` here?

Comment (cbaatz-da) [reply CODEOWNERS]: Done.

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+/extractor/         @digital-asset/extractor
+/navigator/         @digital-asset/navigator
+/web-ide/           @digital-asset/language
+
+# Misc
+/docs/              @digital-asset/docs
+/notices-gen/       @digital-asset/approvers
+/oss-compliance/    @digital-asset/approvers
+
+CHANGELOG           @digital-asset/approvers
```
Comment (garyverhaegen-da) [inline CODEOWNERS]: I'm a bit concerned this is going to actively discourage people from keeping the CHANGELOG up-to-date.

Comment (neil-da) [reply CODEOWNERS]: That's the wrong CHANGELOG, so no one should be keeping it up to date.

Comment (cbaatz-da) [reply CODEOWNERS]: Whether this is a deprecated changelog or not, if it were in use `@digital-asset/approvers` means anyone who has write access can approve it. What would you propose @garyverhaegen-da?

Comment (garyverhaegen-da) [reply CODEOWNERS]: There was no description of what "approvers" mean. If it means everyone, I have no objection.

@neil-da if it's the wrong file, should it be deleted?

Comment (neil-da) [reply CODEOWNERS]: No, it's handy because it points people at the actual changelog.

Comment (cbaatz-da) [reply CODEOWNERS]: @garyverhaegen-da from the email: "Add a @digital-asset/daml-approvers team for everyone who should be able to approve things. I recommend this starts with all code-contributors in the DAML Language, DAML Runtime, and DAML-on-X teams. Other employees can also be added on a case-by-case basis."

**Comment by cbaatz-da** [comment]
We decided to set individual GitHub handles in the CODEOWNERS file and to do so by letting people claim ownership in individual PRs. Thus, this commit contains an initial version of the CODEOWNERS file that guards that file, but leaves the full team with access to other files initially.

Not all files in the root are explicitly listed, but they can be added if someone wants to claim ownership of them.

@bitonic @hurryabit @neil-da ?

**Comment by gerolf-da** [comment]
Do we prefer individual names or github (sub-)teams?

**Comment by cbaatz-da** [comment]
@gerolf-da individuals.

**Pull request merged**